### ISSUE TITLE
Terminal find matches style is inconsistent

### ISSUE THREAD

**Issue by misolori** [microsoft/vscode#147013]
Search and the find widget use the background color to highlight matches:

<img width="539" alt="CleanShot 2022-04-07 at 07 35 35@2x" src="https://user-images.githubusercontent.com/35271042/162224319-6a7581a0-9faf-4880-a801-fdf99083af07.png">

<img width="343" alt="CleanShot 2022-04-07 at 07 36 28@2x" src="https://user-images.githubusercontent.com/35271042/162224463-8dbe74be-ed15-4885-a12e-7a21f21b3a06.png">

And terminal omits the background color, making it inconsistent and unexpected:

<img width="757" alt="CleanShot 2022-04-07 at 07 37 10@2x" src="https://user-images.githubusercontent.com/35271042/162224822-41700f9b-6443-44cc-a726-346705e7eab5.png">

**Comment by meganrogge** [comment]
Yeah we did this because the find background makes the contrast really bad in the terminal. We plan to fix this by layering the decoration behind the text in the future.

**Comment by misolori** [comment]
@meganrogge I understand the struggle of making it look good but we shouldn't start deviating from the pattern as that introduces inconsistencies and creates debt. If we plan to fix this in the future then we shouldn't close out this issue as well for tracking purposes.

**Comment by Tyriar** [comment]
Some things we should call out in the release notes related to this:

- Fixed issue which caused minimum contrast ratio to not work correctly sometimes
- Aligned selection colors with the editor selection since we can now do so without worrying about contrast problems
- Minimum contrast ratio will now "go the other way" when the contrast ratio cannot be met with just reducing or increasing luminance (https://github.com/xtermjs/xterm.js/issues/3720 should be done soon)
- Selection performance improved when gpu acceleration is enabled (maybe not worth mentioning?)

**Comment by Tyriar** [comment]
Created https://github.com/microsoft/vscode/issues/149495 to make sure we remember to add a contrast improvements section 🙂
